src/features/defi/providers/fox-eth-lp/hooks/useFoxEthLiquidityPool.ts

```diff
@@ -0,0 +1,268 @@
+import { Contract } from '@ethersproject/contracts'
+import {
+  ChainAdapter,
+  ethereum,
+  EvmBaseAdapter,
+  EvmChainId,
+  FeeData,
+} from '@shapeshiftoss/chain-adapters'
+import { supportsETH } from '@shapeshiftoss/hdwallet-core'
+import { KnownChainIds } from '@shapeshiftoss/types'
+import IUniswapV2Pair from '@uniswap/v2-core/build/IUniswapV2Pair.json'
+import IUniswapV2Router02ABI from '@uniswap/v2-periphery/build/IUniswapV2Router02.json'
```

Comment:
Also I'd probably just copy this from source instead of installing a new package just for the purpose of importing an ABI
https://github.com/multiswap/contracts/blob/master/contracts/@uniswap/v2-periphery/interfaces/artifacts/IUniswapV2Router02.json